test/e2e/sdk/actorCharge/src/main.mjs

```diff
@@ -0,0 +1,12 @@
+import { Actor, log } from 'apify';
+
+await Actor.init();
+
+const chargeResult = await Actor.charge({
+    eventName: 'foobar',
+    count: 4,
+});
```

Comment:
We could try to utilize the chargeResult in some way. The best that comes to my mind is to store it to KV store and then just load that and check the values